doc/Administration-guide/System-installation/Cluster-setup/index.md

```diff
@@ -10,55 +10,53 @@ sort_idx: 30
-You can find detailed instructions on how to install and configure Zookeeper service and one of supported SQL and NoSQL databases in [Single node installation]({{root_url}}Administration-guide/System-installation/Single-node-installation/#installation-steps) guide. 
-A set of databases (for example MongoDB + PostgreSQL) depends on your particular use case and some useful recommendations you can find in [Planning your deployment]({{root_url}}Administration-guide/System-installation/Planning-your-deployment/) guide.
+```bash
+$ sudo md5sum /usr/lib/kaa-node/keys/bootstrap/public.key
+2df11d4006dbe69c4d208d24f52ea2eb /usr/lib/kaa-node/keys/bootstrap/public.key
+
+$ sudo md5sum /usr/lib/kaa-node/keys/bootstrap/private.key
+cc6f4c598c1ac2cd0c414b54dad953db /usr/lib/kaa-node/keys/bootstrap/private.key
+
+$ sudo md5sum /usr/lib/kaa-node/keys/operations/public.key
+8d9b8d4838ff03ddac52dec97797546c /usr/lib/kaa-node/keys/operations/public.key
+
+$ sudo md5sum /usr/lib/kaa-node/keys/operations/private.key
+f9aab71ef879916d24dfd5200ad94ccd /usr/lib/kaa-node/keys/operations/private.key
+```
 
-In this guide we assume that you had already set up your SQL and NoSQL database clusters, so this tutorial doesn't cover such themes like setting up ones for Cassandra, MongoDB or PostgreSQL. 
-Refer to official [Cassandra](http://docs.datastax.com/en/landing_page/doc/landing_page/current.html), [MongoDB](https://docs.mongodb.com/manual/) and [PostgreSQL](https://www.postgresql.org/docs/) documentation in order to setup corresponding database cluster.
+To refresh the key files, stop the kaa-node service, replace the key files, and start the service again.
 
-MariaDB come with out-of-the-box master-master replication support thus we recommend to use this database in your Kaa cluster. 
-You can find detailed instructions in [MariaDB cluster setup guide]({{root_url}}Administration-guide/System-installation/Cluster-setup/MariaDB-cluster-setup-guide/).
+The Kaa node service requires some third party dependencies, such as SQL and NoSQL databases, and Apache Zookeeper service.
+For more information, see [Architecture overview]({{root_url}}Architecture-overview/).
+A choice of databases (for example, MongoDB + PostgreSQL) depends on your particular use case.
+For database cluster installation, refer to the official [Cassandra](http://docs.datastax.com/en/landing_page/doc/landing_page/current.html), [MongoDB](https://docs.mongodb.com/manual/) and [PostgreSQL](https://www.postgresql.org/docs/) documentation.
 
-> Note: In a cluster you need to connect to databases from external hosts so you need to allow such external connections in corresponding database security configurations and configure firewall rules for database host machine. 
-> Refer to official documentation for corresponding database for security configuration details.
+MariaDB comes with the master-master replication support out of the box, therefore it is recommended that you use this database in your Kaa cluster.
```

Comment:
MariaDB doesn't come with the master-master replication support out of the box. We should remove this sentence.